backend/app/mailers/enrollment_mailer.rb

```diff
@@ -32,6 +32,8 @@ def notification_email
 
   def notification_email_to_instructors
     @enrollment = Enrollment.find(params[:enrollment_id])
+    @demandeur_email = @enrollment.demandeurs.pluck(:email).first
```

Comment:
these variable usage is not very consistent, either:
1. use a single `@demandeur` variable and read its attributes in the template
2. or define 3 variables here.